examples/torchscript/MLproject

```diff
@@ -0,0 +1,25 @@
+name: pytorch_tutorial
```

Comment:
Could we also add a README to the examples/torchscript directory providing an overview of the example / how to run it? It'd also be nice to link to the new example from https://github.com/mlflow/mlflow/blob/master/docs/source/tutorials-and-examples/index.rst

Reply:
Will do